src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -185,8 +192,8 @@ public static int Run(string[] args)
                 var template = language.Templates.FirstOrDefault(t => StringComparer.OrdinalIgnoreCase.Equals(typeValue, t.Name));
                 if (template == null)
                 {
-                    Reporter.Error.WriteLine($"Unrecognized type: {typeValue}".Red());
-                    Reporter.Error.WriteLine($"Available types for {language.Name} :".Red());
+                    Reporter.Error.WriteLine($"{LocalizableStrings.UnrecognizedType}: {typeValue}".Red());
+                    Reporter.Error.WriteLine($"{LocalizableStrings.AvailableTypes} {language.Name} :".Red());
```

Comment:
Here too. typeValue and language.Name should be passed in to {0} from format string.